modules/sb-about-ansible-service-broker.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * applications/service_brokers/installing-ansible-service-broker.adoc
+
+[id='sb-about-ansible-service-broker-{context}']
+= About the {asb-name}
+
+The _{asb-name}_ is an implementation of the Open Service Broker (OSB) API that
+manages applications defined by _Ansible playbook bundles_ (APBs). APBs provide
+a method for defining and distributing container applications in
+{product-title}, consisting of a bundle of Ansible playbooks built into a
+container image with an Ansible runtime. APBs leverage Ansible to create a
+standard mechanism for automating complex deployments.
+
+The {asb-name} follows this basic workflow:
```

Comment:
@shawn-hurley I'm going to merge this PR now. But I did pull in from here down in this file from 3.x (slightly modified) in case you see anything that's off, and I can address in a followup PR.